index.js

```diff
@@ -59,6 +59,8 @@ Auth.prototype.getAuthClient = function (callback) {
     authClient.keyFile = keyFile;
     authClient.email = config.email;
     addScope(null, authClient);
+  } else if (config.apiKey) {
+    googleAuth.fromAPIKey(config.apiKey, addScope);
```

Comment:
It should function the exact same way as the callback signature is the same but (so far as I understand it) API keys are scope-less and, once generated, have permission to talk to any enabled API on the project so the code to actually determine whether a scope authorization is required, in this case, would be redundant.